lib/bump/file_fetchers/ruby/bundler.rb

```diff
@@ -8,6 +8,21 @@ class Bundler < Bump::FileFetchers::Base
         def self.required_files
           %w(Gemfile Gemfile.lock)
         end
+
+        # We currently don't support path-based dependencies, so, we
+        # remove them from the Gemfile entirely.
+        #
+        # We'll resolve the Gemfile without a line (it'll be removed)
+        # from the file, which is not ideal but it'll at
+        # least allow us to resolve what needs to update as opposed
+        # to not resolving anything at all.
+        def files
```

Comment:
The tests for this file are fairly basic and I saw no point in duplicating the test from the base, I reckon it can be improved but I'd rather leave that out as part of this PR, as I'm keen on getting depedabot back online for us.